client/src/components/History/Content/ContentItem.vue

```diff
@@ -176,6 +178,8 @@ export default {
             if (this.isCollection) {
                 return {
                     edit: `collection/edit/${id}`,
+                    showDetails: this.item.job_source_id ? `jobs/${this.item.job_source_id}/view` : null,
+                    reportError: `datasets/error?dataset_id=${id}`, // TODO use proper url for collections
```

Comment:
Is it correct to use the same URL for reporting errors on collections? I couldn't find something like `collections/error`

Reply:
No, that'll be the wrong id ... but in any case this should operate on the job, i think

Reply:
Oh right! I thought it was just using the `creating_job` of the content item even if everything was named as "dataset" but I see the DatasetProvider now... :sweat_smile: 
I guess I should refactor it to accept a job and create another wrapper component that uses the collections provider :thinking:

Reply:
By the way, should I drop the report error button for collections from this PR and add it on dev? If this is somewhat of a new feature?

Reply:
Either way is OK with me, I'll let you decide.